template/{{app_name}}/src/app/[locale]/health/page.tsx

```diff
@@ -1,3 +1,37 @@
+'use client'
+
+import { Member, MembersApi } from "src/api/MembersApi";
+import React, { useEffect, useState } from "react";
+
 export default function Page() {
-  return <>healthy</>;
+  const [members, setMembers] = useState<Member[]>([]);
+
+  const getMembers = async () => {
+    try {
+      const members = await MembersApi.getMembers();
+      setMembers(members);
+      console.log(members);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getMembers();
+  }, []);
```

Comment:
getMembers should be a [dependency](https://react.dev/reference/react/useEffect#specifying-reactive-dependencies)